symplyphysics/laws/electricity/power_from_energy_time.py

```diff
@@ -12,6 +13,17 @@
 # TODO: this law should be derived from definition power_is_energy_derivative
 power, energy, time = symbols('power energy time')
 law = Eq(power, energy / time)
+constant_1, constant_2 = symbols('constant_1 constant_2')
+linear_energy = constant_1 * power_derivative.time +constant_2
+definition_applied = power_derivative.definition.rhs.subs(power_derivative.energy_function(time), linear_energy)
```

Comment:
I think it is a good idea to use local `time` variable. Code will be shorter and you subs it down the code anyway.